### DISCUSSION TITLE
File name on assets selection

### DISCUSSION THREAD

**Discussion by zanhk** [sveltia/sveltia-cms#77] [Ideas]
I think it would be nice to see the filename on the assets selection 

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/b97fd56d-6411-4626-87ab-64012741e981)

like the one in all assets page

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/47c06267-dcc2-451c-b1f7-055906a3d36a)

It would be especcially helpful in cases like this 

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/b315492e-b95c-4563-9220-24e629535e3f)

where you need to know the name of the file as you don't know the association between the image and the name of the file and you have to go look back on the assets page or rely on the search bar

**Comment by kyoshino** [comment]
Makes sense. Will fix it.

**Comment by kyoshino** [comment]
Just shipped [v0.9.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.9.0) with the fix! 🎉

**Reply by zanhk** [reply]
Awesome 👍
